packages/wdio-applitools-service/src/@types/@applitools/eyes-webdriverio.d.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference types="@applitools/visual-grid-client"/>
+
+declare module '@applitools/eyes-webdriverio';
+
+type Region = Eyes.Check.Region | WebdriverIO.Element | string;
+type Frame = WebdriverIO.Element | string;
+
+interface ProxyOptions {
```

Comment:
Can you reuse proxy from `ApplitoolsConfig`? Maybe you can define it there as interface